src/phase-management/phase-rules.json

```diff
@@ -0,0 +1,392 @@
+          "all": [
+            {
+              "fact": "isPastScheduledStartTime",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isOpen",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isClosed",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "submissionCount",
+              "operator": "greaterThanInclusive",
+              "value": 1
+            },
+            {
+              "fact": "isPredecessorPhaseClosed",
+              "operator": "equal",
+              "value": true
+            }
+          ]
+        },
+        "event": {
+          "type": "canOpen"
+        }
+      }
+    ],
+    "IterativeReview": [
+      {
+        "name": "Iterative Review Open",
+        "conditions": {
+          "all": [
+            {
+              "fact": "hasActiveUnreviewedSubmissions",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isOpen",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isPredecessorPhaseClosed",
+              "operator": "equal",
+              "value": true
+            }
+          ]
+        },
+        "event": {
+          "type": "canOpen"
+        }
+      }
+    ],
+    "Appeals": [
+      {
+        "name": "Appeals Open",
+        "conditions": {
+          "all": [
+            {
+              "fact": "isPastScheduledStartTime",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isOpen",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isClosed",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isPredecessorPhaseClosed",
+              "operator": "equal",
+              "value": true
+            }
+          ]
+        },
+        "event": {
+          "type": "canOpen"
+        }
+      }
+    ],
+    "AppealsResponse": [
+      {
+        "name": "Appeals Response Open",
+        "conditions": {
+          "all": [
+            {
+              "fact": "isPastScheduledStartTime",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isOpen",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isClosed",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isPredecessorPhaseClosed",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isPostMortemOpen",
```

Comment:
this looks unnecessary, post mortem is created following registration and submission. If Appeals response is already scheduled then it means that there is no post mortem created in the system.